caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -965,6 +986,8 @@ def initialize_online(self, model_LN=None, T=None):
         self.img_min = img_min
         self.img_norm = img_norm
         if self.params.get('online', 'init_method') == 'bare':
+            # bare is almost like no initialization
+            # cnmf: optional then it runs offline CNMF with offline CNMF init params
```

Comment:
I am confused by the online/offline disparity between the self.params.get and the comment here

Reply:
yea this is confusing, I need to rephrase it. "bare" means no initialization is done at all, "cnmf" means it performs offline CNMF first using the offline CNMF params, and then uses those seeds to perform online CNMF

Reply:
This is exactly the explanation to put in the comment.

Reply:
(it wasn't the "bare" part that I found confusion, it was the online-versus-offline part)